tests/scripts/rebase.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 set -euxo pipefail
 
-# Rebase PRs on master to get latest changes
+SPRAY_NEXT_VERSION=2.$(( ${KUBESPRAY_VERSION:3:2} + 1 ))
```

Comment:
We should use `KUBESPRAY_NEXT_VERSION` for consistency

Reply:
Agreed